### DISCUSSION TITLE
Arquitetura offline first com sincronização  com dados vindo do servidor (firebase)

### DISCUSSION THREAD

**Discussion by ViniciusGularte** [frontendbr/forum#1831] [Perguntas e Respostas]
Olá galera,  to trabalhando num app usando React Native,  e realm db para persistir os dados no mesmo,  tanto ele tanto uma spa ( react também ) iram consumir um banco no firebase, a questão é, qual melhor forma ( se é que existe pq tem varias) de detectar que tem mudanças no banco  (firebase) e sincronizar isto  com o realm db (banco local ) do react native?  no caso com o minimo de requisição possível ao firebase,

**Comment by humbertoromanojr** [comment]
Veja este planejamento, espero que te ajude, porque e material bem estudo para ajudar os Devs

https://blog.rocketseat.com.br/offline-first-em-detalhes/
